src/utils/date.ts

```diff
@@ -0,0 +1,119 @@
+import moment from 'moment'
+import { isDate } from 'lodash'
+
+/**
+   * Converts a Date object to a date string (Month Day, Year) in Pacific timezone.
+   * @param longMonth whether to show long month name (eg, December vs Dec)
+   * @param showWeekday whether to show the weekday name (eg, Thursday)
+   * @example "2021-01-01 07:00:00 GMT" -> "Dec 31, 2020"
+   * @example "2021-01-01 08:00:00 GMT" -> "Jan 1, 2021"
+   */
+export function dateToPacificDate (date: Date, longMonth = false, showWeekday = false): string {
```

Comment:
do we need this file if we have composable ?

Reply:
replaced this and the composable with the util we have in BTR